src/main/resources/db/changelog/changes/02-add-isDeleted-column-to-books-table.yaml

```diff
@@ -0,0 +1,13 @@
+databaseChangeLog:
+  - changeSet:
+      id: add-isDeleted-column
+      author: ST
+      changes:
+        - addColumn:
+            tableName: books
+            columns:
+              - column:
+                  name: is_deleted
```

Comment:
Don't need to create other changelog for this column, add into existing changeset create-book-table